usaspending_api/database_scripts/matviews/universal_transaction_matview.sql

```diff
@@ -25,11 +25,13 @@ SELECT
   "transaction_normalized"."award_id",
   "awards"."category" AS award_category,
   "awards"."total_obligation",
+  "awards"."total_subsidy_cost",
   obligation_to_enum("awards"."total_obligation") AS total_obl_bin,
   "awards"."fain",
   "awards"."uri",
   "awards"."piid",
-  "transaction_normalized"."federal_action_obligation",
+  COALESCE("transaction_normalized"."federal_action_obligation", 0) AS "federal_action_obligation",
```

Comment:
Will postgres maintain the correct type using coalesce?

Reply:
`0` should be able to be interpreted as any valid numeric type, but we could explicitly cast to `NUMERIC(20, 2)` if you believe that would be more comprehensive.

Reply:
Might as well be safe. Do you want to modify all of the existing numeric types to (20,2)?

Reply:
I can do that, will make that update now.